nilearn/_utils/niimg.py

```diff
@@ -29,7 +29,7 @@ def _safe_get_data(img):
     return img.get_data()
 
 
-def load_img(img, dtype=None):
+def load_niimg(img, dtype=None):
```

Comment:
> Renamed it because the input is a niimg but the output is an img.

I am not sure that this is consistent with our naming conventions, and
'smooth_img'.